options/options.go

```diff
@@ -235,6 +235,9 @@ type TerragruntOptions struct {
 	// The file which hclfmt should be specifically run on
 	HclFile string
 
+	// If True then HCL from StdIn must should be formatted.
+	HclFromStdin bool
```

Comment:
I think it should be updated `Clone()` function to copy the value

Reply:
Fixed